pkg/virt-handler/non-root.go

```diff
@@ -74,7 +98,7 @@ func changeOwnershipOfHostDisks(vmiWithAllPVCs *v1.VirtualMachineInstance, res i
 }
 
 func (d *VirtualMachineController) prepareStorage(vmiWithOnlyBlockPVCS, vmiWithAllPVCs *v1.VirtualMachineInstance, res isolation.IsolationResult) error {
```

Comment:
Isn't `vmiWithOnlyBlockPVCS` parameter redundant then? And if that is true, could we rename the parameter `vmiWithAllPVCs` to `vmi` if that makes sense?

Reply:
Indeed, thanks for catching it!

Reply:
Fixed